homeassistant/helpers/entity_registry.py

```diff
@@ -552,6 +559,7 @@ async def async_load(self) -> None:
                     entity_id=entity["entity_id"],
                     icon=entity["icon"],
                     name=entity["name"],
+                    original_device_class=entity["original_device_class"],
```

Comment:
This throws an exception:
```
2021-11-22 17:02:04 ERROR (MainThread) [root] Uncaught exception
Traceback (most recent call last):
  File "/workspaces/core/venv/bin/hass", line 33, in <module>
    sys.exit(load_entry_point('homeassistant', 'console_scripts', 'hass')())
  File "/workspaces/core/homeassistant/__main__.py", line 318, in main
    exit_code = runner.run(runtime_conf)
  File "/workspaces/core/homeassistant/runner.py", line 121, in run
    return loop.run_until_complete(setup_and_run_hass(runtime_config))
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/workspaces/core/homeassistant/runner.py", line 103, in setup_and_run_hass
    hass = await bootstrap.async_setup_hass(runtime_config)
  File "/workspaces/core/homeassistant/bootstrap.py", line 131, in async_setup_hass
    await async_from_config_dict(config_dict, hass) is not None
  File "/workspaces/core/homeassistant/bootstrap.py", line 241, in async_from_config_dict
    await _async_set_up_integrations(hass, config)
  File "/workspaces/core/homeassistant/bootstrap.py", line 535, in _async_set_up_integrations
    await asyncio.gather(
  File "/workspaces/core/homeassistant/helpers/entity_registry.py", line 656, in async_load
    await hass.data[DATA_REGISTRY].async_load()
  File "/workspaces/core/homeassistant/helpers/entity_registry.py", line 562, in async_load
    original_device_class=entity["original_device_class"],
KeyError: 'original_device_class'
```

Reply:
The schema was bumped to 1.2 by https://github.com/home-assistant/core/pull/59912, which was merged last Friday. This PR didn't do an additional bump to 1.3 when adding the device class override. Do we need that?

Reply:
This bumps if it's important to not break nightly: https://github.com/home-assistant/core/pull/60164